Zend/tests/data_classes/assign_op.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
```

Comment:
I would like to see a test with `spl_object_id` confirming the expected CoW semantic like:

```
$o = new data class;
$origId = spl_object_id($o);

$fx = function ($o) use ($origId) {
    assert(spl_object_id($o) === $origId);
    $o->x = 5; // mutate the data class instance to enforce object copy
    assert(spl_object_id($o) === $origId + 1);
};
$fx();

assert(spl_object_id($o) === $origId);
```